conventions/management/commands/change_bailleurs.py

```diff
@@ -0,0 +1,61 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from bailleurs.models import Bailleur
+from programmes.models.models import Programme
+
+
+class Command(BaseCommand):
+    help = "Transfert all Programmes from one Bailleur (old) into another Bailleur (new) using SIREN or SIRET."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--old-bailleur",
+            type=str,
+            required=True,
+            help="SIREN (9 digits) or SIRET (14 digits) of the old Bailleur",
+        )
+        parser.add_argument(
+            "--new-bailleur",
+            type=str,
+            required=True,
+            help="SIREN (9 digits) or SIRET (14 digits) of the new Bailleur",
+        )
+
+    def handle(self, *args, **options):
+        old_id = options["old_bailleur"]
+        new_id = options["new_bailleur"]
+
+        def get_bailleur(identifier: str):
+            if len(identifier) == 14:  # SIRET
+                return Bailleur.objects.filter(siret=identifier).first()
```

Comment:
```suggestion
                return Bailleur.objects.get(siret=identifier)
```

Ca permet de s'assurer qu'il n'y a qu'un seul bailleur avec ce siret et de ne pas faire d'impair ?